docs/user/whats-new.asciidoc

```diff
@@ -90,55 +68,47 @@ image::images/metric-customization.png[Customization options for a metric panel]
-As an admin, you can now assign roles to and edit role permissions on a given space directly from the settings of that space.
+==== Kibana Alerting scalability improvements
+The Kibana Alerting framework delivers enhanced scalability to meet the demands of growing workloads and is available across all Elastic Cloud-hosted deployments. These improvements will provide our customers with 10x the existing capacity to run task-manager tasks -- alerting rules, connector actions, etc. We are observing the following performance improvements on early adopters of the new Kibana Alerting framework:  
 
-Prior to 8.16, you could only do this from the role settings, which was counterintuitive.
+** 10x reduction in alerting task delays
+** 2x increase in task execution throughput on average
+** 50% fewer requests to Elasticsearch, resulting in lower overhead
 
-image::space-settings.png[Editing space settings with new options]
-
-[discrete]
-==== New IP Location processor
-Enhancing location information based on IP addresses just got easier with the new IP Location processor. In addition to the existing free GeoLite offerings from MaxMind, we have integrated with MaxMind’s premium GeoIP databases for users who have licensed MaxMind’s products. If you're an Enterprise Elastic customer, you now have an additional third-party product, IP Info, available for use as well. These additional data sources provide improved options for enriching data with location information associated with IP addresses to improve telemetry and insights. To utilize these features beyond the free MaxMind GeoIP database, you will need to have licensed premium MaxMind products and/or the IP Info database. 
+These customers are now enjoying higher alerting capacity and faster response times without the need for any additional configuration or hardware. Furthermore, these improvements also bring in a new unparalleled scalability for our largest alerting customers, enabling deployments of up to 192 Kibana nodes running alerting rules.
 
-image::images/ip-location-processor.png[The IP Location processor]
+We are very excited to see how these enhancements empower your alerting strategies with faster, more efficient, and scalable performance.
 
 [discrete]
-==== File uploader PDF support
-The file uploader provides a quick way to upload data and start using Elastic. In 8.16, we are improving it to allow you to upload data from PDF files.
+==== Kibana Cases support for file attachments through API actions 
+The latest enhancements to Kibana Cases introduce highly requested functionality for managing security incidents and workflows. Users can now attach files to Kibana Cases via the public API, allowing them to enrich cases with supporting documentation, evidence, and other critical information. This improvement marks a significant enhancement in case management, providing incident response teams with greater flexibility to include all relevant details directly within their cases.
```

Comment:
```suggestion
The latest enhancements to Kibana Cases introduce highly requested functionality for managing security incidents and workflows. You can now use the public API to attach files to cases to enrich them with supporting documentation, evidence, and other critical information. This improvement marks a significant enhancement in case management, providing incident response teams with greater flexibility to include all relevant details directly within their cases.
```